CHANGELOG.md

```diff
@@ -3,6 +3,51 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## v5.12.0
```

Comment:
I was going to release it as 5.11.2, but since we're removing row groping from Pro package I thought justifies releasing a minor version

Reply:
Yeah, I think is better to release as minor. A patch version wouldn't be 100% compatible with the last one.